Makefile

```diff
@@ -26,15 +26,14 @@ include contrib/devtools/Makefile
 build: go.sum
 	go build -mod=readonly ./...
 
-build-sim: go.sum
+build-simd: go.sum
```

Comment:
since this has become the replacement for what gaia was used for, should we provide a install cmd

Reply:
1. build simd
2. ./build/simd